docs/output.md

```diff
@@ -210,27 +218,22 @@ Beta diversity measures the species community differences between samples. Diver
 
 **Output files:**
 
-* `qiime2/phylogenetic_tree/`
-  * `tree.nwk`: Phylogenetic tree in newick format.
-  * `rooted-tree.qza`: Phylogenetic tree in QIIME2 format.
-* `qiime2/diversity/`
-  * `*.txt`: File that describes the rarefaction depth (file name and file contant).
 * `qiime2/diversity/beta_diversity/`
-  * `<method>_distance_matrix-<treatment>/index.html`
-  * `<method>_pcoa_results-PCoA/index.html`
+  * `<method>_distance_matrix-<treatment>/index.html`: Box plots and significance analysis (PERMANOVA).
+  * `<method>_pcoa_results-PCoA/index.html`: Interactive PCoA plot.
     * method: bray_curtis, jaccard, unweighted_unifrac, weighted_unifrac
     * treatment: depends on your metadata sheet or what metadata categories you have specified
 
 #### ANCOM
 
 Analysis of Composition of Microbiomes ([ANCOM](https://www.ncbi.nlm.nih.gov/pubmed/26028277)) is applied to identify features that are differentially abundant across sample groups. A key assumption made by ANCOM is that few taxa (less than about 25%) will be differentially abundant between groups otherwise the method will be inaccurate.
 
-ANCOM is applied to each suitable or specified metadata column for 6 taxonomic levels.
+ANCOM is applied to each suitable or specified metadata column for 5 taxonomic levels (2-6).
```

Comment:
In the ANCOM module is indeed [level 2-6](https://github.com/nf-core/ampliseq/blob/7f4e9cf9b64b710bebc96ad721cb4d9c95b8c87c/subworkflows/local/qiime2_ancom.nf#L25-L31) hard coded, but the meaning of these levels might differ between databases, I think. For DADA2 databases, the tax levels should be stable, however for QIIME2 databases this might vary. Especially when using `--classifier` for custom-trained classifiers. So I hesitate to give here names (i.e. Phylum) instead of numbers (i.e. level 2).